.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: python
 python:
   - "2.7"
-  #- "3.3"
-  - "3.4"
-  - "3.5"
   - "3.6"

```

Comment:
Is support for 3.4 and 3.5 dropped everywhere now?  Or does it break something in particular?

Reply:
3.4 is stuck on an older version of pandas. Can start to fix some of the deprecation warnings by getting rid of that. Dropping 3.5 is a matter of convenience. Reduces time on Travis. With 3.7 out may consider testing 2.7 and 3.7.